SolarSharp.Interpreter/DataTypes/Custom/LuaDictionary.cs

```diff
@@ -1147,10 +1147,17 @@ public bool Remove(TKey key)
                         // we don't clear out entry.key for a few specific reasons;
                         // 1. we want iterators to still be able to get reference to a "freed" key
                         // 2. we should really only have strings as keys
+
+                        // We should clear out the value though for GC efficiency.
+                        // Does this check actually improve performance though?  TODO: Verify.
+#if NETSTANDARD2_1_OR_GREATER || NET8_0_OR_GREATER
                         if (RuntimeHelpers.IsReferenceOrContainsReferences<TValue>())
```

Comment:
No way to call this or pollyfill this in netstandard2.0.

I think it's probably a useless optimization to just have this if here anyways (though it does help with code context of why we are), so I might just remove later.